### PR TITLE
ZEPPELIN-3520. Use the first one as the default value of dynamic form dropdown list

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?
This PR would use the first one as the default value of dynamic form dropdown list. Otherwise user will get error when the return value of null. 

### What type of PR is it?
[Bug Fix]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-3520
### How should this be tested?
* CI pass

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/GUI.java [hunk 1]
```diff
@@ -91,11 +91,15 @@ public Object textbox(String id) {
   }
 
   public Object select(String id, Object defaultValue, ParamOption[] options) {
+    if (defaultValue == null) {
+      defaultValue = options[0].getValue();
```
Comment (felixcheung) [inline zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/GUI.java]: check if options is empty?

Comment (zjffdu) [reply zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/GUI.java]: Fixed

File: zeppelin-integration/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java [hunk 2]
```diff
@@ -674,7 +674,7 @@ public void testMultipleDynamicFormsSameType() throws Exception {
     }
   }
 
-  @Test
+//  @Test
```
Comment (felixcheung) [inline zeppelin-integration/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java]: are these tests broken? where are they disabled?

Comment (zjffdu) [reply zeppelin-integration/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java]: oops, I mix this PR with another PR for ZEPPELIN-3535. Will revert these changes

**Comment by zjffdu** [comment]
Will merge it if no more comments

**Pull request closed without merging**